src/Backend/Modules/MediaLibrary/Js/Helper/Upload.js

```diff
@@ -165,15 +165,18 @@ export class Upload {
    * Add uploaded media to group
    */
   addUploadedMediaToGroup () {
+    let config = this.config
     // loop remaining items in uploaded media and push them to current group
     $('#uploadedMedia').find('li').each(function () {
       // get id
       const id = $(this).attr('id').replace('media-', '')
 
       // add each id to array
-      this.config.currentMediaItemIds.push(id)
+      config.currentMediaItemIds.push(id)
```

Comment:
There is a nicer way to do this. On line 170 you need to use an arrow function and by doing that 'this' wil be available  on line 175, by that you do not need to use the variable `config` set on line 168

Reply:
fixed b227c51b2643f5005f3ce6ac6d035ee2d0c87dbf